conf/application.conf

```diff
@@ -16,6 +15,7 @@
 # This is the main configuration file for the application.
 # ~~~~~
 
+
```

Comment:
have you run precheck before raising this PR ?

Reply:
Of Course